backend/src/models/projects/budget.js

```diff
@@ -0,0 +1,100 @@
+const dbConnection = require("@database/databaseConnection");
+const { knex, dataBaseSchemas } = dbConnection();
+const projectBudgetTable = `${dataBaseSchemas().data}.project_budget`;
+const fiscalYearTable = `${dataBaseSchemas().data}.fiscal_year`;
+const projectDeliverableTable = `${dataBaseSchemas().data}.project_deliverable`;
+const contractsTable = `${dataBaseSchemas().data}.contract`;
+const recoveriesTable = `${dataBaseSchemas().data}.recovery_type`;
+const portfolioTable = `${dataBaseSchemas().data}.portfolio`;
+const clientCodingTable = `${dataBaseSchemas().data}.client_coding`;
+
+const findAllById = (projectId) => {
+  return knex(`${projectBudgetTable} as prb`)
+    .columns(
+      "prb.id",
+      knex.raw("prb.q1_amount::numeric::float8"),
+      "prb.q1_recovered",
+      knex.raw("prb.q2_amount::numeric::float8"),
+      "prb.q2_recovered",
+      knex.raw("prb.q3_amount::numeric::float8"),
+      "prb.q3_recovered",
+      knex.raw("prb.q4_amount::numeric::float8"),
+      "prb.q4_recovered",
+      "fy.fiscal_year as fiscal",
+      "prb.notes",
+      "prd.deliverable_name",
+      knex.raw("prb.detail_amount::numeric::float8"),
+      "rec.recovery_type_name",
+      "prb.stob",
+      "prb.client_coding_id", //TODO Add a lookup in the future.
```

Comment:
We don't have a mockup for this but since the edit form's picker is showing the client_coding.program_area maybe this should too.